sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/CryptographyClientOptions.cs

```diff
@@ -31,9 +31,9 @@ public enum ServiceVersion
             V7_0 = 0,
 
             /// <summary>
-            /// The Key Vault API version 7.1-preview.
+            /// They Key Vault API version 7.1.
```

Comment:
They => The

Reply:
looks like this is in a few other places as well.

Reply:
Oops. Well, that shipped. They're all fixed moving forward. 🤦‍♂️